utils/src/app_storage.rs

```diff
@@ -15,6 +18,10 @@ impl StarboardConfig {
     }
 }
 
+fn one() -> f64 {
+    1.0
+}
```

Comment:
How about using this instead? (and changing line 8 accordingly)

```suggestion
const fn _one() -> f64 {
    1.0
}
```

Reply:
I'm fine with making it const, but why the underscore?

Reply:
It could go either way, but I feel the underscore makes it easier to tell the user is not meant to call the function, but rather the serde macro

Reply:
I decided to not add the underscore and instead tell people through a doc comment to not use the function. (The underscore would prevent the dead code from firing, so it's better to not start identifiers with it)